README.md

```diff
@@ -6,8 +6,9 @@ FireCloud Orchestration Service
 * Github Repository: [https://github.com/broadinstitute/firecloud-orchestration/](https://github.com/broadinstitute/firecloud-orchestration/)
 
 ## IntelliJ Setup
-* Configure the SBT plugin
-* Open this source directory with File -> Open
+* IntelliJ IDEA can be downloaded here : https://www.jetbrains.com/idea/ . It is an 'Intelligent Java IDE'
+* Configure the SBT plugin.  The SBT plugin for IntelliJ IDEA can be downloaded here : https://plugins.jetbrains.com/
```

Comment:
I see what you're saying. How about this:
> Configure the SBT plugin. The SBT plugin can be found by clicking "Browse repositories..." in Plugins settings.

Would that have helped you find it?